src/zcta-county.ts

```diff
@@ -1,23 +1,64 @@
 // Since zip codes can span over multiple counties, this provides a poper taxonomy mapping
 // For the US, converted CSV at
 // http://www2.census.gov/geo/docs/maps-data/data/rel/zcta_county_rel_10.txt
-import zctaCounty from "../public/geo/zcta-county-us.json";
-import gbCountryFipsCode from "../public/geo/gb_counties_fips.json";
+import usZctaCounty from "../public/geo/zcta-county-us.json";
+import auZctaCounty from "../public/geo/zcta-county-au.json";
+import gbCountyFipsCode from "../public/geo/gb-counties-fips.json";
+import auCountyFipsCode from "../public/geo/au-counties-fips.json";
 
-const countyZctaMap = zctaCounty.reduce((acc,r)=> {
+const usCountyZctaMap = usZctaCounty.reduce((acc,r)=> {
     acc.set(r.geoid,r.zcta)
     return acc;
 }, new Map<string, Array<string>>());
 
-const gbCountryFipsCodeMap = gbCountryFipsCode.reduce((acc, r) => {
+const auCountyZctaMap = auZctaCounty.reduce((acc, r) => {
+    acc.set(r.geoid, r.zcta)
+    return acc;
+}, new Map<string, Array<string>>());
+
+/**
+ * Mapping of county names as appear in data CSV file to county IDs as they appear in 
+ * the map shapefile for the UK
+ */
+const gbCountyFipsCodeMap = gbCountyFipsCode.reduce((acc, r) => {
+    acc.set(r.county_name, r.county_fips_code)
+    return acc;
+}, new Map<string, string>());
+
+/**
+ * Mapping of county names as appear in data CSV file to county IDs as they appear in 
+ * the map shapefile for Australia
+ */
+const auCountyFipsCodeMap = auCountyFipsCode.reduce((acc, r) => {
     acc.set(r.county_name, r.county_fips_code)
```

Comment:
If you use state_name+county_name as the key in the map then you won't need to use a separate lookup table to handle duplicates.